### PR TITLE
📊 covid: cases and deaths

### PR DESCRIPTION
**Pull request by lucasrodes**
[[tracking issue](https://github.com/owid/owid-issues/issues/1619)]

In this PR, I add the code necessary to update the COVID cases and deaths data directly via ETL. In the past, we have done it using the [covid-19-data](github.com/owid/covid-19-data) repository, which was created pre-ETL.

## Tasks
- [x] Cases/Deaths
  - [x] Preliminary import
  - [x] Update charts
  - [x] Set up regular updates (pending buildkite addition)
- [x] Docs: Add guideline on how to automate a regular update. To review it, run `mkdocs serve` and then visit http://localhost:8000/projects/etl/guides/auto-regular-updates/.

## Future
The pipeline in covid-19-data will continue to be executed, but the generated indicators will not be used on Grapher. Instead, they only power data hosted in the repository. The plan is to replace these scripts with API-access to ETL indicators via `owid-grapher-py`.

### CODE REVIEW

File: etl/steps/data/garden/covid/latest/cases_deaths.py [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+    # Convert Table to DataFrame
+    df = pd.DataFrame(tb)
+
+    # Period-aggregate cases and deaths
+    cases_colname = f"{prefix}_cases"
+    deaths_colname = f"{prefix}_deaths"
+    df[[cases_colname, deaths_colname]] = (
+        df[["country", "new_cases", "new_deaths"]]
+        .groupby("country")[["new_cases", "new_deaths"]]
+        .rolling(window=periods, min_periods=periods - 1, center=False)
```
Comment (Marigold) [inline etl/steps/data/garden/covid/latest/cases_deaths.py:270]: I'll see if I can make these work with Table (but feel free to merge it like this).

**Review by Marigold — APPROVED** [review]
Code looks good! Thanks for writing up the docs.

[This chart](http://staging-site-covid-migrate/grapher/total-covid-cases-deaths-per-million) shows a small jump down for confirmed cases (which should be increasing). Any idea why?

Have you figured out how are we going to update data in the covid repo?

**Comment by lucasrodes** [comment]
Thanks, @Marigold! On the covid-19-data repo I plan to use `owid-catalog-py` to read the dataset, and then will export the necessary files.

**Comment by lucasrodes** [comment]
>[This chart](http://staging-site-covid-migrate/grapher/total-covid-cases-deaths-per-million) shows a small jump down for confirmed cases (which should be increasing). Any idea why?

Good catch, Mojmir! This comes from the denominator (i.e. 'population'), which changes from year-on-year. I'll work on a fix.

**Comment by lucasrodes** [comment]
CI/CD seems to be not working as expected. See [discussion](https://owid.slack.com/archives/C0193RW5E2J/p1721988802412659)

I branched out and will review charts again here: https://github.com/owid/etl/pull/3025

**Pull request closed without merging**